packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -68,47 +71,47 @@ export default class QuanticStandaloneSearchBox extends NavigationMixin(
     }));
   }
 
+  get standaloneEngineId() {
+    return `${this.engineId}_standalone`;
+  }
+
   connectedCallback() {
-    registerComponentForInit(this, this.engineId);
+    registerComponentForInit(this, this.standaloneEngineId);
   }
 
   renderedCallback() {
-    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+    initializeWithHeadless(this, this.standaloneEngineId, this.initialize.bind(this));
   }
 
-  get actualSearchBox() {
-    if (window.location.href.includes(this.redirectUrl)) {
-      return this.searchBox;
+  @wire(CurrentPageReference)
+  handlePageChange() {
```

Comment:
Have you tested this on LWR too?
Both types of community, I am not sure, might react differently to the page navigation.

Reply:
Do you have an org i could use to test this quickly?

Reply:
Any new scratch org you make should be able to have the LWR community template to make a new one.
(My scratch orgs probably are all expired otherwise I would have one for ya)

Reply:
ah yes i didnt know they'd updated the starting template. It now has a header and body which makes it super easy to test with. The behaviour seems to be as desired in LWR